### DISCUSSION TITLE
cant login - network error

### DISCUSSION THREAD

**Discussion by Bowen73** [amidaware/tacticalrmm#1040] [Q&A] [answered]
Somehow i appear to be locked out of my TacticalRMM.  Using the credentials i always have but now i just get 'network error'  flashing up.

mine is the only login so cannot try another login and there is no 'forgot password' link on the login page so cant try that.  I have it hosted on google compute so i have access to the backend but cant login to the front end system.

What can i try/do to get access

Thanks

**Comment by dinger1986** [comment, marked as answer]
https://docs.tacticalrmm.com/management_cmds/#reset-a-users-password

Read the top of the page to make sure you are in the python environment before running the commands.

**Reply by Bowen73** [reply]
Thanks.  everything else i found told me to login to change the password :-s

I've done that and it's changed the password.  I went back and tried to login and I'm still getting 'network error' when i try to login.  no idea how it can all of a sudden stop working as it was working fine.

**Reply by dinger1986** [reply]
How long have you had the install?

**Reply by Bowen73** [reply]
I setup an ssh profile on my laptop on 16th dec, so will be around then and was all working fine until recently

**Reply by Bowen73** [reply]
when i was able to log in and put in the 2fa code it would briefly flash up network error before actually logging in.  No idea if that's normal or relevant but thought its worth a mention, just incase

**Reply by dinger1986** [reply]
Are your ssl certs needing updated? I would imagine so if using default ones

**Reply by Bowen73** [reply]
ive not put ssl certs on the compute instance, if that's what you mean.  i have SSL on my main domain but the redirect for the sub domain to the compute instance. unless it was part of the install instructions.  i cant remember :-s

**Reply by Bowen73** [reply]
just checked.  i must have done as i have the acme-challenge on my dns

**Reply by Bowen73** [reply]
i did wonder when i tried log in i was on HTTP not https how would i update it

**Reply by Bowen73** [reply]
just had a look and found this so will give it a go.  will have to set myself a reminder for 3 months as I'll be in the same situation.  Is there a way to automatically do it?  i can across one but it was for a docker install
https://docs.tacticalrmm.com/update_server/#keeping-your-lets-encrypt-certificate-up-to-date

**Reply by dinger1986** [reply]
It shouldn't work on http at all ssl is forced on.

https://docs.tacticalrmm.com/update_server/#keeping-your-lets-encrypt-certificate-up-to-date

**Reply by dinger1986** [reply]
Check unsupported in docs for auto renewing/using own certs

**Reply by silversword411** [reply]
Wow, sounds complicated and definitely not part of a supported installation process. #WelcomeWizard

**Reply by Bowen73** [reply]
i'll have a look, thanks.  I cant take up to 24hrs for DNS to update so will try again in a little while as its normally ok after an hour or 2.  no idea if SSL is forced on.    I'll take a look at the unsupported to see if i can get it automated.  no biggy if not, now i know what it is.

thanks for your assistance, you're a star!

**Reply by dinger1986** [reply]
Use google dig to check it

**Reply by Bowen73** [reply]
yeah its showing the right txt thats on the domain that i just set but still get the same error.  https is still crossed out too and i dont get the 2fa popup.
i thought i was on to a winner then too haha

**Reply by dinger1986** [reply]
So did the cert renew? Did you run `update.sh --force`. 

It's all in the docs

**Reply by Bowen73** [reply]
rookie mistake :/ after i ran it i flipped to the page to refresh and tried login... just gon back to ssh and said the command dint exist, i was in the wrong directory! doh!  

just re-ran it and actually waited instead of being an impatient tit to make sure...  just tried to log in and got the 2fa popup.  success.  
Thank you!! I'm a happy bunny again now hahah

**Reply by dinger1986** [reply]
Lol glad it's sorted.

It's almost always something simple with TRMM that's wrong the devs have done such a good job!
